### DISCUSSION TITLE
Welcome to components Discussions!

### DISCUSSION THREAD

**Discussion by phatdev12** [Folody-Team/components#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phatdev12** [comment]
> ##  Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together muscle.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

An environment for everyone to grow

**Reply by kirigizakazutovn** [reply]
yes
